### PR TITLE
[Issue #6843] Remove use of some magic numbers

### PR DESCRIPTION
**Pull request by burner761**
this Request seeks to make the code more readable by reducing the use of magic numbers which can confuse people who want to edit the code in the future.

### CODE REVIEW

File: geonode/static/geonode/js/cookie-law-info/cookie-law-info-public.js [hunk 1]
```diff
@@ -480,10 +480,11 @@ var CLI=
...
     {
     	var cli_elm=this.bar_elm;
-	    var cli_win=jQuery(window);
-	    var cli_winh=cli_win.height()-40;
+		var cli_win=jQuery(window);
+		var cli_winh=cli_win.height()-40; 
 	    var cli_winw=cli_win.width();
-	    var cli_defw=cli_winw>400 ? 300 : cli_winw-30;
+		var cli_defw=cli_winw>400 ? 300 : cli_winw-30; 
+
```
Comment (mtnorthcott) [inline geonode/static/geonode/js/cookie-law-info/cookie-law-info-public.js:487]: Revert indentation here.

File: geonode/static/geonode/js/ol-2.13/lib/deprecated.js [hunk 2]
```diff
@@ -25,7 +25,9 @@
  * @requires OpenLayers/Popup/Anchored.js
  * @requires Rico/Corner.js
  */
-
+XMLisOpen = 1;
+XMLhasReceivedHeaders = 2
+XMLisLoaded = 4;
```
Comment (mtnorthcott) [inline geonode/static/geonode/js/ol-2.13/lib/deprecated.js:30]: Inconsistent use of semicolons here. Also could probably declare these as `const`

File: geonode/static/geonode/js/ol-2.13/lib/deprecated.js [hunk 3]
```diff
@@ -1035,6 +1038,8 @@ OpenLayers.Util.extend(OpenLayers.Tile.prototype, {
      * bounds - {<OpenLayers.Bounds>} 
      */
     getBoundsFromBaseLayer: function(position) {
+        var fullRot = 360;
```
Comment (mtnorthcott) [inline geonode/static/geonode/js/ol-2.13/lib/deprecated.js:1041]: Define as a `const`

File: geonode/static/geonode/js/ol-2.13/lib/deprecated.js [hunk 4]
```diff
@@ -1326,9 +1331,10 @@ OpenLayers.Control.MouseDefaults = OpenLayers.Class(OpenLayers.Control, {
      * Zoombox function. 
      */
     zoomBoxEnd: function(evt) {
+        mouseSensitivity = 5
```
Comment (mtnorthcott) [inline geonode/static/geonode/js/ol-2.13/lib/deprecated.js:1334]: Define as a `const`

File: geonode/static/geonode/js/ol-2.13/lib/deprecated.js [hunk 5]
```diff
@@ -1373,6 +1379,7 @@ OpenLayers.Control.MouseDefaults = OpenLayers.Class(OpenLayers.Control, {
     onWheelEvent: function(e){
     
         // first determine whether or not the wheeling was inside the map
+        var scrollSensitivity = 120;
```
Comment (mtnorthcott) [inline geonode/static/geonode/js/ol-2.13/lib/deprecated.js:1382]: Define as a `const`

File: geonode/utils.py [hunk 6]
```diff
@@ -82,6 +82,7 @@
     SplitResult
 )
 
+maxExtent = 20037508.34;
```
Comment (mtnorthcott) [inline geonode/utils.py:85]: Global constants in Python should be capitalised.

File: geonode/utils.py [hunk 7]
```diff
@@ -112,6 +113,7 @@
 
 logger = logging.getLogger("geonode.utils")
 
+fullRot = 360;
```
Comment (mtnorthcott) [inline geonode/utils.py:116]: Global constants in Python should be capitalised.

**Review by mtnorthcott — CHANGES_REQUESTED** [review]
Mainly style changes.

**Comment by t-book** [comment]
@mtnorthcott This looks good for me, ready for merge from your side?

**Review by mtnorthcott — APPROVED** [review]

**Comment by mtnorthcott** [comment]
@t-book Looks good to me

**Comment by t-book** [comment]
nice thanks @mtnorthcott 

- Code looks good to me
- Tested live, no regression spotted.

Merged.

**Pull request merged**